src/functions/guild/editEmbed.js

```diff
@@ -0,0 +1,58 @@
+module.exports = async d => {
+  const data = d.util.aoiFunc(d);
+  var [channelID = d.channel?.id, messageID = d.message?.id, index = 1, option = 'description', content] = data.inside.splits;
+ index = index - 1;
+const channel = await d.util.getChannel(d, channelID);
+if (!channel) return d.aoiError.fnError(d, 'channel', {}, {inside: data.inside});
+const message = await d.util.getMessage(channel, messageID);
+if (!message) return d.aoiError.fnError(d, 'message', {}, {inside: data.inside});
+function isNumber(value) {
+  return typeof value === "number" && !Number.isNaN(value);
+}
+if (!isNumber(index)) return d.aoiError.fnError(d, 'custom', {}, 'Invalid Index');
+if (!content) return d.aoiError.fnError(d, 'custom', {}, 'No Content Provided');
+var embed = new d.embed(message.embeds[index]);
+if(option.startsWith("field[")) {
+const fi = option.charAt(6) - 1;
+option = option.slice(0, 6) + fi + option.slice(7); 
+option = option.slice(0, 5) + "s" + option.slice(5); 
+} else if(option.startsWith("field")) {
+const fi = option.charAt(5) - 1;
+option = option.slice(0, 5) + fi + option.slice(6); 
+option = option.slice(0, 5) + "[" + option.slice(5);
+option = option.slice(0, 7) + "]" + option.slice(7);
+option = option.slice(0, 5) + "s" + option.slice(5); 
+} else if(option == "timestamp") {
```

Comment:
don't do this, this is actually unreadable, check parsers src for reference

Reply:
Ah i am sorry, I don't know why I did this, when I could have just used .substring(). Well I committed a new update for it. Please look into it.